packages/relay/src/routes/post.ts

```diff
@@ -137,13 +138,24 @@ async function createPost(
             calldata
         )
 
-        const post = await db.create('Post', {
-            content: content,
-            cid: cid,
-            epochKey: epochKeyProof.epochKey.toString(),
-            epoch: epoch,
-            transactionHash: hash,
-            status: 0,
+        const epochKey = epochKeyProof.epochKey.toString()
+
+        // after post data stored in DB, should add 1 to epoch key counter
+        await addActionCount(db, epochKey, epoch, (txDB) => {
+            txDB.create('Post', {
```

Comment:
maybe we could reuse this post object instead query post again

Reply:
I think we don't have to return post & epoch to the frontend. As you can see the `createPost` function on `useCreatePost` hook, it only retrieve the transaction hash to wait for this transaction on-chain.

https://github.com/social-tw/social-tw-website/blob/7eca64f5f9d8e722c52f3166ec1c7488a677cae1/packages/frontend/src/hooks/useCreatePost.ts#L43C3-L59